Control/TMSITable.cpp

```diff
@@ -819,13 +819,21 @@ vector< vector<string> > TMSITable::tmsiTabView(int verbosity, bool rawFlag, uns
 				}
 			} else if (header == "CREATED" || header == "ACCESSED") {
 				// Print seconds as a time value.
-				row.push_back(prettyAge(now - query.getResultInt(col)));
+				if (rawFlag) {
+					row.push_back(format("%d", query.getResultInt(col)));
```

Comment:
Shouldn't this also be `now - query.getResultInt(col)`? Seems like raw should not change the semantics of the field.